CHANGELOG.md

```diff
@@ -5,13 +5,13 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
-- 
+-
 
 ### Changed
-- 
+- **cf-buttons:** Convert button links to actual buttons.
```

Comment:
Super minor: Should this say "Change button links in Docs..." since this isn't changing anything about the cf-buttons code?